modules/minimum-ibm-power-system-requirements.adoc

```diff
@@ -19,7 +19,7 @@ You can install {product-title} version {product-version} on the following IBM h
 [discrete]
 == Operating system requirements
 
-* One instance of an IBM POWER8 or POWER9 processor-based system
+* One instance of an IBM Power8, Power9, or Power10 processor-based system
```

Comment:
Hi! I know you didn't make changes to the areas above and below this line, but leaving them as is kind of stands out to me:

The Hardware requirements and the three items in Operating system requirements following this Line 22 both use digits instead of words for the number. Line 22 uses the word. Being right in the middle of these items, it looks awkward. Would you change the digits to words, please? Thanks!